cypress/integration/4_findings.spec.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { PLUGIN_NAME } from '../support/constants';
+import sample_document from '../fixtures/sample_document.json';
+import sample_index_settings from '../fixtures/sample_index_settings.json';
+import sample_field_mappings from '../fixtures/sample_field_mappings.json';
+import sample_detector from '../fixtures/sample_detector.json';
+
+describe('Findings', () => {
+  const ruleTags = ['low', 'windows'];
+
+  before(() => {
+    cy.deleteAllIndices();
+
+    // Visit Findings page
+    cy.visit(`${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/findings`);
+
+    // create test index, mappings, and detector
+    cy.createIndex('cypress-test-windows', sample_index_settings);
+    cy.createDetector(sample_detector);
+    cy.createAliasMappings('cypress-test-windows', 'windows', sample_field_mappings, true);
```

Comment:
we should create the mapping before creating detector to be consistent with how it works in the UI

Reply:
I will fix and test.  BRB

Reply:
done and pushed.